### ISSUE TITLE
abusing trial not working still says that it expired

### ISSUE THREAD

**Issue by GuestedKun** [EugeneSunrise/reWASD#7]
I did the tutorial spoofed my hwid and everything and still. the thing doesn't work my trial keeps saying expired do i need to use an specified version? i have the latest one and nope it doens't work

if u guys know a way to fix iit please tell me haha

**Comment by kuresc** [comment]
You just need to do 1, 6, 7 steps. It works you can skip other steps.

**Comment by PreppyBrain** [comment]
> You just need to do 1, 6, 7 steps. You can skip other steps.

Can you help to automate that steps?

**Comment by Emilia33** [comment]
> You just need to do 1, 6, 7 steps. You can skip other steps.

i did this but still not working

**Comment by PreppyBrain** [comment]
> > You just need to do 1, 6, 7 steps. You can skip other steps.
> 
> i did this but still not working

how did you uninstalled rewasd? What software you use to do this?

**Comment by Emilia33** [comment]
> > > You just need to do 1, 6, 7 steps. You can skip other steps.
> > 
> > 
> > i did this but still not working
> 
> how did you uninstalled rewasd? What software you use to do this?

you can use revo uninstaller

**Comment by FlyerMan31MTL** [comment]
> You just need to do 1, 6, 7 steps. You can skip other steps.

Hi, which process are you referring to? can you link a thread?

**Comment by RedDot-3ND7355** [comment]
I've just cracked the latest version with radial and all.
I got the whole packet json response for valid stuff down.
Haven't checked your method yet but I assume it's related to the check whether the feature is paid for or not (int == 3)

Mine perma-activates it on startup.
Tested all features working as well.

But here's a preview of what I've done :)

![image](https://github.com/EugeneSunrise/reWASD/assets/6676924/4ffe785b-6f9e-46ad-9e60-0bc8b5c918c8)
